apps/src/javalab/constants.js

```diff
@@ -3,13 +3,21 @@
  * Check their use in the Javabuilder repo before editing.
  */
 import {makeEnum} from '@cdo/apps/utils';
+import msg from '@cdo/javalab/locale';
 
 export const CsaViewMode = {
   NEIGHBORHOOD: 'neighborhood',
   CONSOLE: 'console',
   THEATER: 'theater'
 };
 
+export function getUnsupportedMiniAppMessage(miniAppType) {
```

Comment:
Wasn't sure what the best place for this helper function was. It's not a constant, so it feels a bit weird to put in this file, but it is directly related to CsaViewMode (aka mini app type) and is used externally in separate files, so this felt like one possible common spot for it. Definitely open to other ideas, so let me know if there's a better place

Reply:
I'd be in favor of making a new file named something like `utils` that this and potentially other methods could go into

Reply:
Good idea, I'll go ahead and do that